server/bundler/loader.js

```diff
@@ -111,6 +111,7 @@ function splitTemplate( path, section ) {
 		'		if ( ! LoadingError.isRetry() ) {',
 		'			LoadingError.retry( ' + JSON.stringify( section.name ) + ' );',
 		'		} else {',
+		'			console.error(error);',
```

Comment:
Do you plan to leave this debug here?

Reply:
Yes, I do.
It's [this commit](https://github.com/Automattic/wp-calypso/pull/17539/commits/7a77a3f1dcf71d29ff08719e67d1e8658809e7ec) which basically fixes a bug where you would not see when a chunk would fail to be evaluated (when loaded), for instance if there is a failed import or any kind of runtime error babel cannot see on build time.
cc @ockham